bytes/pyproject.toml

```diff
@@ -57,23 +55,16 @@ opentelemetry-instrumentation-requests = "^0.41b0"
 opentelemetry-instrumentation = "^0.41b0"
 opentelemetry-instrumentation-dbapi = "^0.41b0"
 pydantic-settings = "^2.0.3"
+python-multipart = "^0.0.9"
```

Comment:
This shouldn't be a top-level dependency as far as I can see.

Reply:
That's what I thought and removed it at first, but apparently FastAPI needed it so I reverted that action

Reply:
I saw before that it was a dependency of FastAPI... but it is an optional dependency, so it it not installed by defautl.

Reply:
I know, but we use it. https://github.com/minvws/nl-kat-coordination/actions/runs/7922423908/job/21629977395

And I'm not sure if there's an alternative installation of FastAPI (e.g. `pip install fastapi[with-multipart]`)